kafka/test_kafka.py

```diff
@@ -23,6 +32,22 @@ def __run_test_script(self, name):
         self.assert_instance_command(
             name, "bash {}".format(self.TEST_SCRIPT_FILE_NAME))
 
+    def __submit_pyspark_job(self, cluster_name):
+        if self.getImageVersion() >= pkg_resources.parse_version("2.1"):
+            verify_script = self.KAFKA_VERIFY_SCRIPT_PYTHON3
+            python_version = self.PYTHON3_VERSION
+        else:
+            verify_script = self.KAFKA_VERIFY_SCRIPT_PYTHON2
```

Comment:
verify_kafka_running_py3.py and verify_kafka_running.py have a lot in common. I think we should merge then, and pass in parameters or detect different image versions inside the script.

Reply:
based on the dataproc image version both the scripts are merged and tested locally.